x-pack/test/functional/es_archives/alerts/data.json

```diff
@@ -1041,3 +1041,63 @@
      }
   }
 }
+
+{
+  "type":"doc",
+  "value":{
+     "id":"alert:734def70-f8b1-11ec-b8e0-c99fab63aaae",
+     "index":".kibana_1",
+     "source":{
+        "alert":{
+           "name":"Test rule with snoozeEndTime",
+           "alertTypeId":".index-threshold",
+           "consumer":"alerts",
+           "params" : {
+             "aggType" : "count",
+             "termSize" : 5,
+             "thresholdComparator" : ">",
+             "timeWindowSize" : 5,
+             "timeWindowUnit" : "d",
+             "groupBy" : "all",
+             "threshold" : [
+               0
+             ],
+             "index" : [
+               ".test-test-test"
+             ],
+             "timeField" : "@timestamp"
+           },
+           "schedule":{
+              "interval":"1m"
+           },
+           "enabled":true,
+           "actions":[
+              
+           ],
+           "throttle":null,
+           "notifyWhen" : "onActiveAlert",
+           "apiKeyOwner" : "elastic",
+           "apiKey" : "y86eFOuG06GNd4p/90Zh4V4C+4wF8rFHiLcOeNqHQFFAbagYCXIuggq/N/AB0HBncCsWrh5PsCs4RiX53+LP2o+oLtoQkdTK9m+iv0BnzP6aavoypIudKtnElN3SotI3XpIRm7c3MEdM/C/zZXHgCmn4UMExhkulOO1S/Q//FwvIr2/SKw/mp8Vx5qP41hqDX4YFfSMoFkPSLw==",
```

Comment:
I think there is a correct-er way to generate this data so that the api key can be decrypted by the functional test server. This is just one I've copied from local data and it gives a decryption error during the test :(